src/Microsoft.Restier.EntityFramework/Model/ModelProducer.cs

```diff
@@ -124,17 +121,19 @@ private ModelProducer()
         }
 
         private static IEdmEntityType CreateEntityType(
-            MetadataWorkspace efModel, EntityType efEntityType,
-            EdmModel model, out List<EdmStructuralProperty> concurrencyProperties)
+            MetadataWorkspace efModel,
+            EntityType efEntityType,
+            EdmModel model,
+            Dictionary<MetadataItem, IEdmElement> elementMap,
```

Comment:
yeah, those are private implementations.. I think Dictionary is fine, but considering we may replace the impl with ConcurrencyDictionary here, I'm to update it to IDictionary now.